recipes/deps/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "deps" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ESSS/{{ name }}.git
+  git_rev: v{{ version }}
```

Comment:
Can you use a tarball instead of the rev/tag/commit? That way we have an "oficial" release, we can check the checksum (although I get that git is probably safer than that :wink:), but we also avoid closing the whole repo :smile:

Reply:
Sure... I just don't know how git tarballs are stable (sometimes they change the hash because of changes in the repo metadata), but I'll change it here

Reply:
Yeah, that is quite annoying b/c there was no change to the actual tarball :unamused: 

You can upload a tarball and host in the "tag page" with git to avoid that.
(But then you would need to jump some extra hoops, sorry.)